cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_angular_client_app/templates/src/app/model/model.ts.ftl

```diff
@@ -1,9 +1,5 @@
 <#compress>
-<#assign seen_properties=[]>
 <#list model.properties as property>
-  <#if seen_properties?seq_contains(property.type) == false>
-      <#if property.isEntity> export { ${property.type?cap_first} } from './${property.type}'; </#if>
-      <#assign seen_properties = seen_properties + [property.type]>
-  </#if>
+    <#if property.isEntity> export { ${property.type?cap_first} } from './${property.type}'; </#if>
```

Comment:
Please revert the changes from this file. Most probably you are using an old version.